src/Navigation/Navigation.js

```diff
@@ -13,11 +13,12 @@ const Navigation = ({loggedIn, setLoggedIn}) => {
   }
 
   return (
-    <section>
-      <p className="welcome-message">Welcome User!</p>
+    <section className="nav-header">
+      <Link to="/" className="header-link">
+        <h1 className="page-title">National Park Service Service</h1>
+      </Link>
       <nav>
-        {loggedIn ? <button onClick={handleLoginNavigate}>Logout</button> : <button onClick={handleLoginNavigate}>Login</button>}
-        {loggedIn ?? <button onClick={visitSaved}>Saved Parks</button>}
+        {loggedIn ? <section className="banner"><p className="welcome-message">Welcome User!</p><button onClick={visitSaved}>Saved Parks</button><button onClick={handleLoginNavigate}>Logout</button></section> : <button onClick={handleLoginNavigate}>Login</button>}
```

Comment:
i think this looks really clean. good job thinking of the nav component, i hadnt thought to do that